### PR TITLE
Init jwplayer passing DOM ref instead of id

### PR DESCRIPTION
**Pull request by cavez86**
Initializing jwplayer passing DOM ref allows to use this Component inside a web component

### CODE REVIEW

**Comment by haintwork** [comment]
I got this issue, `window.jwplayer is not a function` is it fixed here?

**Comment by ellell** [comment]
It seems this update actually does break some functionality for us. In some cases we're using the playerId to access the player sdk for some custom functionality. Any way we can make the this update backward compatible, by opting in to use the dom ref but still defaulting to playerId?

**Comment by cavez86** [comment]
I pushed an update to set the ref id. I have still problems with googleima ads inside web components but otherwise all seems working.

File: src/react-jw-player.jsx [hunk 1]
```diff
@@ -64,34 +66,36 @@ class ReactJWPlayer extends Component {
...
     }
   }
   componentWillUnmount() {
-    removeJWPlayerInstance(this.props.playerId, window);
+    removeJWPlayerInstance(this.videoRef, window);
   }
   _initialize() {
     const { playerId, useMultiplePlayerScripts } = this.props;
 
+    this.videoRef.id = playerId;
```
Comment (ellell) [inline src/react-jw-player.jsx]: Any specific reason to set this here rather than attaching the id to the element directly in the render method?

Comment (cavez86) [reply src/react-jw-player.jsx]: No, edited this

**Comment by danmakenoise** [comment]
Code looks good to me, I can't verify it works on my end right now, thought. If it's working for you guys, I trust ya!

**Pull request merged**